test/lib/ansible_test/_data/sanity/code-smell/runtime-metadata.py

```diff
@@ -72,9 +72,10 @@ def main():
 
     plugin_routing_schema = Any(
         Schema({
-            ('deprecation'): Any(deprecation_schema),
+            ('deprecation'): Any(deprecation_tombstoning_schema),
+            ('tombstone'): Any(deprecation_tombstoning_schema),
```

Comment:
I guess it doesn't make sense to have both at the same time, but let's sort this out later. It's more important to get a basic validation.